### DISCUSSION TITLE
Transcoding always uses libx264

### DISCUSSION THREAD

**Discussion by trissi1987** [photoprism/photoprism#2389] [Q&A] [answered]
Hello
I have been using Photoprism for a short time. 
I would like to use my iGPU to transcode videos. Photoprism always use libx264.
I can't find any errors in the log either.
I have already made the following settings:

- `PHOTOPRISM_INIT: gpu tensorflow`
- `PHOTOPRISM_FFMPEG_ENCODER: intel`
- `PHOTOPRISM_FFMPEG_BUFFERS: 64`

also

- `PHOTOPRISM_INIT: intel-graphics tensorflow-amd64-avx2`
- `PHOTOPRISM_FFMPEG_ENCODER: h264_qsv`
- `PHOTOPRISM_FFMPEG_BUFFERS: 64`

Can anyone help me here? I use the following things:

Unraid (6.9.2) with Docker
CPU: J4105 with Quick Video Sync
`/dev/dri` exists in Container.

#1337
#2222

**Comment by graciousgrey** [comment, marked as answer]
Did you sign up as [sponsor](https://photoprism.app/membership) already ? Some of the env variables you mention are sponsor features as indicated in the [docs](https://docs.photoprism.app/getting-started/config-options/#file-converters).

**Reply by trissi1987** [reply]
Thank you very much for pointing that out. I must have overlooked it.

**Reply by trissi1987** [reply]
> Did you sign up as [sponsor](https://photoprism.app/membership) already ? Some of the env variables you mention are sponsor features as indicated in the [docs](https://docs.photoprism.app/getting-started/config-options/#file-converters).

Thank You...now it works!
Only now it seems that the transcoding videos that have been started are not being played. Is it possible to clear the cache?

**Reply by trissi1987** [reply]
`photoprism reset`
